### PR TITLE
[Activiti/Activiti#4401] [AAE-16065] add displayName in project

### PR DESCRIPTION
**Pull request by indrayanganguly**

### CODE REVIEW

File: activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-jpa/src/main/java/org/activiti/cloud/services/modeling/entity/ProjectEntity.java [hunk 1]
```diff
@@ -53,6 +54,9 @@ public class ProjectEntity extends AuditableEntity<String> implements Project<St
 
     private String name;
 
+    @Column(name = "disp_name", nullable = false)
```
Comment (jsokolowskii) [inline activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-jpa/src/main/java/org/activiti/cloud/services/modeling/entity/ProjectEntity.java]: For now the display name should be nullable (at least until it is also developed on FE side).

File: activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-liquibase/src/main/resources/config/modeling/liquibase/master.xml [hunk 2]
```diff
@@ -8,46 +8,46 @@
   <changeSet author="aae-modeling"
              id="h2-schema-creation" dbms="h2">
     <sqlFile dbms="h2"
-             encoding="utf8"
```
Comment (jsokolowskii) [inline activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-liquibase/src/main/resources/config/modeling/liquibase/master.xml]: Could you please revert the formatting changes to the file? The git history and the PR would be cleaner.

**Review by jsokolowskii — CHANGES_REQUESTED** [review]
2 small comments, other than that LGTM. Thanks!

File: activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-jpa/src/main/java/org/activiti/cloud/services/modeling/entity/ProjectEntity.java (hunk 1 above)
Comment (indrayanganguly) [reply activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-jpa/src/main/java/org/activiti/cloud/services/modeling/entity/ProjectEntity.java]: Yes, done.

File: activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-liquibase/src/main/resources/config/modeling/liquibase/master.xml (hunk 2 above)
Comment (indrayanganguly) [reply activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-liquibase/src/main/resources/config/modeling/liquibase/master.xml]: Done.

**Review by jsokolowskii — APPROVED** [review]
LGTM

**Pull request merged**